py2pack/templates/fedora.spec

```diff
@@ -53,9 +53,13 @@ rm -rf %{buildroot}
 {%- if doc_files %}
 %doc {{ doc_files|join(" ") }}
 {%- endif %}
-{%- for script in scripts %}
```

Comment:
You could just use `{%- for script in scripts|default([], true) %}` I think

Reply:
Oh, that's nice! Didn't know it existed in Jinja